source/_components/sensor.weightedaverage.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "Weighted Average Sensor"
+description: "Instructions on how to add weighted average sensors within Home Assistant."
+date: 2019-06-01 01:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```

Comment:
doubt if you can use the HA logo for this

Reply:
OK, what should I use instead? Can I just skip the logo?

Reply:
you can skip it but it doesn't look nice in the components overview. Pherhaps just a picture of a scale?